extension/headerssetter/go.mod

```diff
@@ -0,0 +1,36 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/extension/headerssetter
+
+go 1.17
```

Comment:
Can you double-check other modules? I think this was recently bumped to 1.18.

Reply:
Update the version and squashed commits.